tests/phpunit/tests/post.php

```diff
@@ -756,4 +756,23 @@ public function test_use_block_editor_for_post() {
 		$this->assertTrue( use_block_editor_for_post( $restless_post_id ) );
 		remove_filter( 'use_block_editor_for_post', '__return_true' );
 	}
+
+	/**
+	 * Ensure get_post_meta does not try to get metadata from a revision
+	 *
+	 * @ticket 58763
+	 * @covers ::get_post_meta
+	 */
+	public function test_get_post_meta() {
```

Comment:
I found this test passes before the patch. To test the fix it should fail before the patch is applied.